pftools/python/parflow/tools/builders.py

```diff
@@ -353,28 +454,77 @@ def load_txt_content(self, txt_content):
         return self
 
     def assign(self, old=None, new=None, mapping=None):
-        if old != new:
+        """Method to assigning subsurface properties of one unit to another
+
+        Args:
+            old=None (str): Source unit with existing parameters
+            new=None (str): Target unit to which the parameters
+                            from old will be mapped
+            mapping=None (dict): Dictionary that includes the old units as keys
+                and new units as values.
+        """
+        # assigning subsurface properties of one unit to another
+        if isinstance(new, list):
+            for item in new:
+                self.assign(old, item)
+
+        elif old != new:
             data = self.output[old]
             self.output[new] = data
 
         if mapping is not None:
             for old, new in mapping.items():
-                if isinstance(new, list):
-                    for item in new:
-                        self.assign(old, item)
-                else:
-                    self.assign(old, new)
+                self.assign(old, new)
 
         return self
 
-    def load_default_properties(self):
+    def load_default_properties(self, database='conus_1'):
+        """Method to load one of several default property databases.
+
+        Args:
+           database='conus_1': default database - options are:
+           'conus_1': soil/rock properties from Maxwell and Condon (2016)
+           'washita': soil/rock properties from Little Washita script
+           'freeze_cherry': soil/rock properties from Freeze and Cherry (1979)
+           Note: Freeze and Cherry only has permeability and porosity
+        """
+        database_file = 'ref/subsurface_' + database + '.txt'
+
         default_prop_file = os.path.join(
-            os.path.dirname(__file__), 'ref/default_subsurface.txt')
-        self.load_txt_file(default_prop_file)
+            os.path.dirname(__file__), database_file)
+
+        if exists(default_prop_file):
+            self.load_txt_file(default_prop_file)
+            print('#'*80)
+            print('# Loaded database:')
+            for item in self.table_comments:
+                print(item)
+            print('#' * 80)
+        else:
+            print('#' * 80)
+            print(f'# {database} database not found. Available databases include:')
+            for root, dirs, files in os.walk(os.path.dirname(__file__) + '/ref/'):
+                for name in files:
+                    if name.startswith('subsurface'):
+                        print(f'# - {name} (use argument "{name[11:-4]}")')
```

Comment:
If this `11` and `4` correspond to the lengths of some strings, it might be more maintainable to do something like:

```python
[len('<str>'):-len('<str2>')]
```

instead.